woocommerce/woocommerce-gateway-stripe
# Feature/1690 add informative overlay

Fixes #1690

### Description
The changes in this PR implements an informative overlay to provide users some information about Stripe. This overlay is only visible once the user enables UPE and visits `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout` for the first time. This is a one time overlay and disappears forever when user click on the `Got it` button or `Cross` icon or outside of the modal.

# Testing instructions
- Enable UPE
- Visit `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout`. The overlay should appear
- Dismiss the message. The overlay should disappear
- Refresh the page. The overlay should never appear again.
